decidim-core/lib/decidim/content_parsers/context.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Decidim
+  module ContentParsers
+    # Class to inject context information to content parsers
+    #
+    # @example How to create a context
+    #   context = Decidim::ContentParsers::Context.new(
+    #     current_organization: a_decidim_organization
+    #   )
+    class Context
```

Comment:
Why not make this a Hash, and keep passing the hash around? 🤔

Reply:
I think this way is better for documentation and you can improve the context adding helper/convenience methods to the class in the future.

The only drawback is that the developer is not allowed to pass arbitrary data to the parser, but the context can be improved to accept dynamic attributes or add an `extra_data` attribute, if needed in the future.

Anyway, if the @decidim/lot-core decision is to use a plain hash as a context, is fine for me, I'll do the necessary changes to have this PR merged ASAP because it is blocking other team mates in @decidim/lot-mods.

What is the decision about it?

Reply:
I'd go with a Hash then, if you don't mind. Also, note there are CodeClimate issues and failing tests

Reply:
Ok, I'll make the change. Also, note that this PR is not ready for review, it have the `in-progress` label. I usually check if all the checks pass and then assign the `in-review` label. 

Don't get me wrong, I only ask to have the opportunity to request the review when I think that the PR is ready, and then, I'll be ready for the battle, hahaha. 😆

Reply:
No problem at all! 😄